pkg/plugins/policies/meshaccesslog/api/v1alpha1/schema.yaml

```diff
@@ -53,10 +53,14 @@ properties:
                                       type: string
                                     value:
                                       type: string
+                                  required:
+                                    - key
+                                    - value
                                   type: object
                                 type: array
                               omitEmptyValues:
                                 default: false
+                                description: the below optional is a temporary workaround
```

Comment:
ugh, this line got into the schema, you can try making it the same line like:
```go
OmitEmptyValues bool `json:"omitEmptyValues"` // optional annotation is a temporary workaround
```
or I guess just remove the comment since the solution is going to land shortly after